dist/pwa-mat/service-worker.js

```diff
@@ -7,7 +7,13 @@ importScripts("precache-manifest.2bc800e8d7cdc5d90f3a80e5f7b27eb9.js", "https://
  */
 workbox.core.setCacheNameDetails({prefix: 'tptea'})
 self.workbox.precaching.precache(self.__precacheManifest)
-workbox.routing.registerRoute(new RegExp('/api'), workbox.strategies.staleWhileRevalidate())
+workbox.routing.registerRoute(({url}) => {
```

Comment:
find a way to just simply use RegExp, no need to search for words in url.search params